x/crosschain/keeper/cctx.go

```diff
@@ -48,7 +47,7 @@ func (k Keeper) SetCctxAndNonceToCctxAndInTxHashToCctx(ctx sdk.Context, cctx typ
 			Tss:       tss.TssPubkey,
 		})
 	}
-	if cctx.CctxStatus.Status == types.CctxStatus_Aborted && cctx.GetCurrentOutTxParam().CoinType == common.CoinType_Zeta {
+	if cctx.CctxStatus.Status == types.CctxStatus_Aborted && cctx.InboundTxParams.CoinType == common.CoinType_Zeta {
```

Comment:
Yes, OutTxParams.CoinType should always be equal to InTxParams.CoinType.
This change is done in this pr , so that its easier to remove the duplicate Cointype in the future